README.md

```diff
@@ -10,12 +10,17 @@ Give a Lunch helps feed hungry healthcare workers while supporting local busines
 
 The site is built with Gatsby started with the [Ueno Starter](https://www.gatsbyjs.org/starters/ueno-llc/ueno-gatsby-starter/).
 
+- Node v10.17.0 (You may use nvm to manage multiple Node versions)
 - Typescript
 - Sass
 
-## Development
+## Development Setup
 
 1. Get [access to airtable](https://airtable.com/invite/l?inviteId=invB6xFse6TbB1BmT&inviteToken=0055f625782dd335f34548c9d6e1ffbc71464e35b1c1549018e10ad9c9b23c47)
 2. Copy your Airtable API key (Account -> Generate API Key)
 3. `yarn install`
 4. `export AIRTABLE_API_KEY=${your_api_key} yarn dev`
+
+## Running
+
+- `gatsby develop`
```

Comment:
These updates are great! My only suggestion is to remove step 4 above since `yarn dev` technically runs `gatsby develop`